_posts/2016-11-07-recurse-center-week-6.md

```diff
@@ -58,15 +58,17 @@ The last few weeks I've been focused on the most sociable people at RC. The ones
 
 Last week I stepped back from all the socializing. I mainly interacted with the people who were still chatting to me despite my drop in energy levels. It was interesting to see how that shifted who I interacted with.
 
-I started to appreciate all the interesting people who aren't necessarily sharing social events 24/7. I also realized how many people I really enjoy talking to despite never having really interacted outside of RC. It felt really lovely ☺️.  
+I started to appreciate all the interesting people who aren't necessarily sharing social events 24/7. I also realized how many people I really enjoy talking to despite never having really interacted outside of RC. That felt really lovely 🙂.  
 
 #### **Less people to think about.**
 
-I feel a sense of relief from having half the people leave. For 6 weeks I invested all I could into this large group of people. Now that 1/2 have left what happens next is out of my control. If we formed a connection, we'll stay in touch. If not, we won't. Detaching from outcomes is nice.
+I feel a sense of relief from having half the people leave. For 6 weeks I invested all I could into this large group.
+
+Now that half have left, what happens next is out of my control. If we formed a connection, we'll stay in touch. If not, we won't. Detaching from outcomes is nice.
 
 At the same time, there's still a large group of wonderful people here. People who I can continue getting to know for another 6 weeks.
 
-When the new people arrive, I'll feel a lot less pressure to know everyone.
+With the new people, I'll feel a lot less pressure to know everyone.
```

Comment:
Ahaha. Let's think about this mathematically!

In my brain I have a control algorithm. It's job is to make decisions trend me towards happiness.

One of the metrics for happiness is `SocialSatisfaction`. Social satisfaction is calculated using a parameter called `friendsCount` and `friendsDepth`. Max social satisfaction is say `100`. So we can calculate it as such:

```
SocialSatisfaction = (friendsCount...
```

Gotta pair, will explain later 😝